crypto/context.c

```diff
@@ -38,8 +38,7 @@ static void do_default_context_deinit(void)
 }
 DEFINE_RUN_ONCE_STATIC(do_default_context_init)
 {
-    return OPENSSL_init_crypto(0, NULL)
-        && context_init(&default_context)
+    return context_init(&default_context)
         && OPENSSL_atexit(do_default_context_deinit);
```

Comment:
I've realised that if `OPENSSL_atexit()` is called, a `OPENSSL_init_crypto(OPENSSL_INIT_BASE_ONLY, NULL)` must be made first...  so instead of removals, I'll just change the first argument

Reply:
Wait, that wasn't true, so never mind (OPENSSL_init_crypto(OPENSSL_INIT_BASE_ONLY,NULL)` does *nothing* regarding atexit processing anyway)